page.css

```diff
@@ -1,5 +1,4 @@
 ---
-layout: null
```

Comment:
Does this still work?? If so, very cool. (I thought the layout tag was required)